presto-iceberg/src/test/java/com/facebook/presto/iceberg/rest/TestIcebergSmokeRestNestedNamespace.java

```diff
@@ -339,10 +339,12 @@ public void testView()
         assertUpdate(session, "CREATE VIEW view_orders AS SELECT * from orders");
         assertQuery(session, "SELECT * FROM view_orders", "SELECT * from orders");
         assertThat(computeActual("SHOW CREATE VIEW view_orders").getOnlyValue())
-                .isEqualTo(format("CREATE VIEW iceberg.\"%s\".view_orders AS\n" +
-                        "SELECT *\n" +
-                        "FROM\n" +
-                        "  orders", schemaName));
+                .isEqualTo(format("CREATE VIEW iceberg.\"%s\".view_orders SECURITY %s  AS\n" +
```

Comment:
Need to add extra space after %s

Reply:
instead can you fix the formatting from the SqlFormatter so that there is only one space before/after.